docs/articles/streams/integration.md

```diff
@@ -406,6 +406,55 @@ The numbers in parenthesis illustrates how many calls that are in progress at
 the same time. Here the downstream demand and thereby the number of concurrent
 calls are limited by the buffer size (4) of the `ActorMaterializerSettings`.
 
+### Integrating with Observables
+
+Starting from version 1.3.2, Akka.Streams offers integration with observables - both as possible sources and sinks for incoming events. In order to expose Akka.Streams runnable graph as an observable, use `Sink.AsObservable<T>` method. Example:
+
+```csharp
+IObservable<int> observable = Source.From(new []{ 1, 2, 3 })
+    .RunWith(Sink.AsObservable<int>(), materializer);
+```
+
+In order to use an observable as an input source to Akka graph, you may want to use `Source.FromObservable<T>` method. Example:
+
+```csharp
+await Source.FromObservable(observable, maxBufferCapacity: 128, overflowStrategy: OverflowStrategy.DropHead)
+    .RunForEach(Console.WriteLine, materializer);
+```
+
+You may notice two extra parameters here. One of the advantages of Akka.Streams (and reactive streams in general) over Reactive Extensions in notion of backpressure - absent in Rx.NET. This puts a constraint of rate limiting the events incoming form upstream. If an observable will be producing events faster, than downstream is able to consume them, source stage will start to buffer them up to a provided `maxBufferCapacity` limit. Once that limit is reached, an overflow strategy will be applied. There are several different overflow strategies to choose from:
```

Comment:
Is “in notion” correct? Sounds a bit strange to me or should it have been “is notion” ?